src/styles/main.styl

```diff
@@ -1,10 +1,10 @@
 @require 'generic/normalize.styl'
 @require 'elements/defaults.styl'
-@require 'utilities.styl'
-@require 'text.styl'
+@require '~styles/utilities.styl'
+@require '~styles/text.styl'
 @require 'settings/breakpoints'
 @require '~styles/variables.styl'
-@require 'palette.styl'
+@require '~styles/palette.styl'
```

Comment:
To me ~ meant node_modules ? Is it the case here ?

Reply:
Explained via mattermost : ~ mean any directory in resolve.modules since we use webpack resolver.